docs/source/common-configuration-patterns.md

```diff
@@ -0,0 +1,169 @@
+# Common Configuration Patterns
+
+## Global config
+
+Most of the configuration files have a lot in common. This tends to be especially true for fields describing managers:
+- `area`
+- `storage`
+- `logging`
+
+From our experience it is sometimes easiest to create a so called *global configuration*, which contains all such fields.
+
+```
+{  // global_config.json
+  "area": {
+    ...
+  },
+  "storage": {
+    ...
+  },
+  "logging": {
+    ...
+  }
+}
+```
+
+This is then used in pipeline configurations.
+
+```
+{ // export.json
+  "pipeline": "eogrow.pipelines.export_maps.ExportMapsPipeline",
+  "**global_config": "${config_path}/global_config.json",
+  "feature": ["data", "BANDS"],
+  "map_dtype": "int16",
+  "cogify": true,
+  ...
+}
+```
+
+This keeps pipeline configs shorter and more readable. One can also use multiple such files, for instance one for each manager. This makes it easy to have pipelines that work on different resolutions, where we just switch between `"**area_config": "${config_path}/area_10m.json"` and `"**area_config": "${config_path}/area_30m.json"`.
+
+How fine grained you want to get is usually project-specific. Spreading it too thinly makes it harder to follow what precisely will be in the end config.
+
+### Adjusting settings from the global config
+
+In some cases the settings from a global config (or from a different config file that you are importing) need to be overridden. Imagine that a pipeline produces a ton of useless warnings, and you only wish to ignore them for that specific pipeline.
+
+```
+{ // export.json
+  "pipeline": "eogrow.pipelines.export_maps.ExportMapsPipeline",
+  "**global_config": "${config_path}/global_config.json",
+  "logging": {
+    "capture_warnings": false
+  },
+  "feature": ["data", "BANDS"],
+  "map_dtype": "int16",
+  "cogify": true,
+  ...
+}
+```
+
+The processed configuration will have all the logging settings from `global_config.json`, except for `"capture_warnings"`. See config language rules for config joins.
```

Comment:
Is there a link for this?

Reply:
we'd need to check how to link to other MD-as-RST documents. Perhaps that sentence can just go out?

Reply:
Is there a link for this?

Reply:
managed to find the magic combo, link works now.

Reply:
cool, will merge on test success unless you have additional plans